go/cmd/vtctldclient/command/vreplication/mount/mount.go

```diff
@@ -0,0 +1,175 @@
+		Use:                   "Mount [command] [command-flags]",
+		Short:                 "Mount is used to link an external Vitess cluster in order to migrate data from it.",
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"mount"},
+		Args:                  cobra.ExactArgs(1),
+	}
+)
+
+var mountOptions struct {
+	TopoType   string
+	TopoServer string
+	TopoRoot   string
+}
+
+var register = &cobra.Command{
+	Use:                   "register",
+	Short:                 "Register a mount",
+	Example:               `vtctldclient --server localhost:15999 Mount Register --topo-type etcd2 --topo-server localhost:12379 --topo-root /vitess/global ext1`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Register"},
+	Args:                  cobra.ExactArgs(1),
+	RunE:                  commandRegister,
+}
+
+func commandRegister(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountRegisterRequest{
+		TopoType:   mountOptions.TopoType,
+		TopoServer: mountOptions.TopoServer,
+		TopoRoot:   mountOptions.TopoRoot,
+		Name:       cmd.Flags().Arg(0),
+	}
+	_, err := common.GetClient().MountRegister(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Mount %s registered successfully\n", req.Name)
+	return nil
+}
+
+var unregister = &cobra.Command{
+	Use:                   "unregister",
+	Short:                 "Unregister a mount",
+	Example:               `vtctldclient --server localhost:15999 Mount Unregister ext1`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Unregister"},
+	Args:                  cobra.ExactArgs(1),
+	RunE:                  commandUnregister,
+}
+
+func commandUnregister(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountUnregisterRequest{
+		Name: args[0],
+	}
+	_, err := common.GetClient().MountUnregister(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Mount %s unregistered successfully\n", req.Name)
+	return nil
+}
+
+var show = &cobra.Command{
+	Use:                   "show",
+	Short:                 "Show attributes of a mount",
+	Example:               `vtctldclient --server localhost:15999 Mount Show ext1`,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Show"},
+	Args:                  cobra.ExactArgs(1),
+	RunE:                  commandShow,
+}
+
+func commandShow(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MountShowRequest{
+		Name: args[0],
+	}
+	resp, err := common.GetClient().MountShow(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+	data, err := json.Marshal(resp)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s\n", string(data))
+	return nil
+}
+
+var list = &cobra.Command{
+	Use:                   "list",
+	Short:                 "List all mounts",
+	Example:               `vtctldclient --server localhost:15999 Mount List`,
```

Comment:
And here.

I also think that it would be nice to note WHAT the mounts are throughout the code. Could just be e.g. "List all remote topo server mounts"